app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -67,6 +68,10 @@ def confirm
       end
     end
 
+    def passkey_not_backed_up
+      @continue_path = next_setup_path || after_mfa_setup_path
+    end
```

Comment:
Is there any way we could shoehorn this action name into a standard Rails one? (`new`, `index`, `show` etc?) I was very confused by this

Reply:
I'm increasingly feeling like we should just drop the route / view code from this pull request, since we're not actively pursuing it anymore. The authenticator data flags are what I think we can focus on in this pull request. What do you think @mdiarra3 ?

Reply:
Im fine with that, I can remove it